Library/Homebrew/development_tools.rb

```diff
@@ -19,6 +19,12 @@ def installed?
       which("clang") || which("gcc")
     end
 
+    def installation_instructions
+      "Install Clang or brew install gcc"
+    end
+    alias_method :custom_installation_instructions,
+                 :installation_instructions
```

Comment:
Sorry, again missed the point. Should have been looking at the diff further down first …